lib/ship_quiet_logistics/commands/processor.rb

```diff
@@ -11,20 +11,28 @@ def initialize(blackboard, queue)
 
         @config = ShipQuietLogistics.configuration
       end
-
       def call
         messages_count.times do
           document = next_document
           next if !document || document.nil?
+          case document_type document.namespace
+            when 'RMADocument'
+              config.process_rma_handler.(document)
```

Comment:
I think we are missing the configuration option for `process_rma_handler` look for `process_shipment_handler` to see where it's done file is `ship_quiet_logistics.rb`.